float64 kinetic_energy
float64 potential_energy
float64 total_energy
float64 timestamp
